CHANGELOG

```diff
@@ -1,5 +1,110 @@
 This file contains a summary of changes to the Oppia code base.
 
+v3.0.5 (02 Dec 2020)
+------------------------
+Contributor Pages
+* Fix part of #10687: Make it possible to get the top 5 suggestions that have been waiting longest for review per reviewer. (#10860)* Fix #10955: Make the updates to the CommunityContributionStatsModel transactional in the PopulateContributionStatsOneOffJob (#10956)
+* Fix #10687: Make it possible to notify Contributor Dashboard reviewers that there are suggestions waiting for review. (#10943)
+* Fix translation count issue for the exploration opportunity models (#11290)
+* Add job to delete invalid suggestion models on server (#11274)
+* Add job to rename the ExplorationOpportunitySummaryModel properties (#11093)
+* Fix #10707, #10969: Notify admins about bottlenecks on the Contributor Dashboard causing a longer review turnaround time (#10963)
+
+Cloud NDB Migration
+* Improve code health of test_utils (#11080)
+
+Bug fixes
+* Fix #11200: fix browser detection mismatch between frontend and backend (#11211)
```

Comment:
```suggestion
* Fix #11200: Fix browser detection mismatch between frontend and backend (#11211)
```

For consistency with other items in this changelog.

Reply:
Done.